*** tests/A-01-24-M-ok.out	2024-10-03 22:30:36.182224800 +0100
--- tests/A-01-24-M-ok.outhyp	2024-10-05 12:02:22.011537300 +0100
***************
*** 16,24 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|V3|Tétano|0|E1,E2,E3
! VACINA|V4|Tétano 2|0|E1,E2
! VACINA|V5|Tétano 3|0|E1,E2
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
--- 16,22 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Nenhuma vacina registrada.
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
