*** tests/A-01-07-M-ok.out	2024-10-03 22:28:52.230920200 +0100
--- tests/A-01-07-M-ok.outhyp	2024-10-05 12:02:18.277933200 +0100
***************
*** 16,22 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|A1|Farrusco|E1|VOID|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 16,22 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Nenhum animal registado.
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
